Gopkg.toml

```diff
@@ -55,12 +55,46 @@ required = [
   name = "github.com/hashicorp/vault"
   version = "v0.9.3"
 
+[[constraint]]
+  name = "sigs.k8s.io/controller-runtime"
+  # temporarily, before release
+  branch = "master"
+
 # Use the GitHub mirror of goautoneg to save issues with bitbucket being down
 [[override]]
   name = "bitbucket.org/ww/goautoneg"
   revision = "a547fc61f48d567d5b4ec6f8aee5573d8efce11d"
   source = "https://github.com/munnerz/goautoneg.git"
 
+### Overrides to make CR happy (isn't dependency management wonderful?)
+[[override]]
+  name = "k8s.io/api"
+  version = "kubernetes-1.13.2"
+
+[[override]]
+  name = "k8s.io/apimachinery"
+  version = "kubernetes-1.13.2"
+
+[[override]]
+  name = "k8s.io/client-go"
+  # Use the specific revision in client-go that fixes a regression in the
+  # testing package
+  # We can remove this once 1.14 is released.
+  source = "https://github.com/munnerz/client-go.git"
+  revision = "6761e17484a547b310ba2418dc56b655901a73aa"
+  # version = "kubernetes-1.13.2"
+
+[[override]]
+  name = "k8s.io/apiextensions-apiserver"
+  version = "kubernetes-1.13.2"
```

Comment:
I don't think the duplicated overrides are required? We pin to these same versions and depend on them directly already, so dep should force the chosen versions here ❓